### PR TITLE
Renamed ROT to TurnSpeed

### PR DESCRIPTION
**Pull request by reaperrr**
...on `Mobile`, `Aircraft`, `Turreted` and `ThrowsParticle`.

It was mentioned long time ago that we should get rid of abbreviations.

For the record: `TurnSpeed` got the most agreement on IRC (because it matches `Speed`), so I will not rename it again.

### CODE REVIEW

File: OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs [hunk 1]
```diff
@@ -599,6 +599,15 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				if (engineVersion < 20160301)
+				{
+					// Renamed ROT -> TurnSpeed
+					if (node.Key == "ROT")
+					{
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs]: Unnecessary brackets.

**Comment by obrakmann** [comment]
:+1:

File: OpenRA.Mods.Common/Traits/Turreted.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ public class TurretedInfo : ITraitInfo, UsesInit<TurretFacingInit>, Requires<Bod
 	{
 		public readonly string Turret = "primary";
 		[Desc("Rate of Turning")]
```
Comment (GraionDilach) [inline OpenRA.Mods.Common/Traits/Turreted.cs]: Shouldn't this be updated then as well?

Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/Turreted.cs]: Yeah, well spotted.

Comment (GraionDilach) [inline OpenRA.Mods.Common/Traits/Turreted.cs]: There is another instance of this though. So yeah, all descs should be unified as part of this PR imo.

**Comment by reaperrr** [comment]
Updated descriptions.

Also changed `Turreted`'s `InitialFacing` to 0, to match `Mobile`.

**Comment by abcdefg30** [comment]
:+1:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/c4e1ab43e44df23bc425ace06b3952b3f7b457fc...a24659a4bef92ad3a28350400825ca3a038add16)

**Pull request merged**